modules/ossm-security-mtls.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_user_guide/ossm-security.adoc
+
+[id="ossm-security-mtls_{context}"]
+= Enabling mutual Transport Layer Security (mTLS)
+
+Mutual Transport Layer Security (mTLS) is a protocol where two parties authenticate each other at the same time. It is the default mode of authentication in some protocols (IKE, SSH) and optional in others (TLS).
+
+MTLS can be used without changes to the application or service code. The TLS is handled entirely by the service mesh infrastructure and between the two sidecar proxies.
+
+By default, {ProductName} is set to permissive mode, where the sidecars in {ProductShortName} accept both plain-text traffic and connections that are encrypted using mTLS. If a service in your mesh is communicating with a service outside the mesh, strict mTLS could break communication between those services. Use permissive mode while you migrate your workloads to {ProductShortName}.
+
+== Enabling strict mTLS across the mesh
+
+If your workloads do not communicate with services outside your mesh and communication will not be interrupted by only accepting encrypted connections, you can enable mTLS across your mesh quickly. Set `spec.istio.global.mtls.enabled` to `true` in your ServiceMeshControlPlane resource. The operator creates the required resources.
+
+[source,yaml]
+----
+apiVersion: maistra.io/v1
+kind: ServiceMeshControlPlane
+spec:
+  istio:
+    global:
+      mtls:
+        enabled: true
+----
+
+[id="ossm-security-mtls-sidecars-incoming-services_{context}"]
+=== Configuring sidecars for incoming connections for specific services
+
+You can also configure mTLS for individual services or namespaces by using a `Policy` instead of `ServicMeshPolicy`. 
```

Comment:
Not sure if you want to leave `ServiceMeshPolicy` here, as it's configured through mtls.enabled=true

Reply:
@rcernich Ok, good catch. I removed that and added a line about making sure to change the control plane namespace. However, I have two questions, in Istio, policies are deprecated. Is that still our strategy? And how does this policy apply to a specific service? I don't see a place to add the service name.

Should the <CONTROL_PLANE_NAMESPACE> variable be <Namespace>?